Stardust Error only on PieCharts

## 🐛 Bug report

Unable to get piecharts to render with a freshly generated app

### Steps to Reproduce

1. Followed this guide https://qlik.dev/tutorials/build-a-simple-web-app
2. Added piechart to configure.js like so..
![image](https://user-images.githubusercontent.com/43914884/89834049-16768280-db30-11ea-95a6-cac2d71cda64.png)
![image](https://user-images.githubusercontent.com/43914884/89834149-3c9c2280-db30-11ea-8885-c426149126fb.png)


3. Attempted a render in my index.js using on-the-fly and prebuilt vizualizations with their ID.

### Expected behavior

Expected the visualization to show up.

### Actual behavior

Instead visualization is blank and we get a stardust error in console.

`stardust.js:20 Error: Invalid stardust hook call. Hooks can only be called inside a visualization component.
    at getHook (stardust.dev.js:23037)
    at useInternalContext (stardust.dev.js:23071)
    at useOptions (stardust.dev.js:23760)
    at J_ (sn-pie-chart.js:57)
    at component (sn-pie-chart.js:57)
    at stardust.js:20
    at tryCatch (stardust.dev.js:298)
    at Generator.invoke [as _invoke] (stardust.dev.js:518)
    at Generator.prototype.<computed> [as next] (stardust.dev.js:351)
    at n (stardust.js:7)`

### Versions

- "@nebula.js/cli": "^1.0.0",
    "@nebula.js/cli-build": "^1.0.0",
    "@nebula.js/sn-action-button": "^1.6.0",
    "@nebula.js/sn-bar-chart": "^0.4.0",
    "@nebula.js/sn-kpi": "^0.3.2",
    "@nebula.js/sn-line-chart": "^0.4.0",
    "@nebula.js/sn-mekko-chart": "^1.0.4",
    "@nebula.js/sn-pie-chart": "0.4.0",
    "@nebula.js/stardust": "1.0.0",
    "@nebula.js/theme": "^0.6.0",
    "enigma.js": "^2.7.1",
    "hammerjs": "^2.0.8",
    "parcel-bundler": "^1.12.4"
- Platform: Windows
- Browser: Chrome, Edge, Firefox

### Additional context
![image](https://user-images.githubusercontent.com/43914884/89833829-bb449000-db2f-11ea-9b45-cfd70d194ce2.png)

Note this only happens with PieCharts, line charts/barcharts/etc. all work fine.

Replicated on 3 machines so far.

--- a/commands/create/templates/mashup/src/configure.js
+++ b/commands/create/templates/mashup/src/configure.js
@@ -1,4 +1,4 @@
-import { embed } from '@nebula.js/stardust/dist/stardust';
+import { embed } from '@nebula.js/stardust';
 
 import barchart from '@nebula.js/sn-bar-chart';
 

